attestation-agent/attestation-agent/src/config/kbs.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2024 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use serde::Deserialize;
+
+use super::aa_kbc_params;
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct KbsConfig {
+    /// URL Address of KBS.
+    pub url: String,
+
+    /// Cert of KBS
+    pub cert: Option<String>,
+}
+
+impl Default for KbsConfig {
+    fn default() -> Self {
+        let aa_kbc_params =
+            aa_kbc_params::get_params().expect("No aa_kbc_params specified in kernel cmdline");
```

Comment:
get_params will use multiple methods
```suggestion
            aa_kbc_params::get_params().expect("failed to set aa_kbc_params");
```

Reply:
Thanks. I assume you mean `get`